packages/twenty-front/src/modules/workflow/utils/getStepOutputSchemaKey.ts

```diff
@@ -0,0 +1,4 @@
+export const getStepOutputSchemaKey = (
```

Comment:
we should likely prefix everything with workflow
getWorkflowStepSchemaFamilyStateKey